README.md

```diff
@@ -1,25 +1,42 @@
 # DevOps Apprenticeship: Project Exercise
```

Comment:
In this file you're missing:

- Notes on setting up the Trello Config
- Instructions on how to run the tests